src/WebJobs.Script/Host/BlobLeaseManager.cs

```diff
@@ -115,25 +118,40 @@ private void ProcessLeaseTimerTick(object state)
         {
             try
             {
+                DateTime requestStart = DateTime.UtcNow;
                 if (HasLease)
                 {
                     await _lockBlob.RenewLeaseAsync(new AccessCondition { LeaseId = LeaseId });
-                    _traceWriter.Verbose("Host lock lease renewed.");
+                    _lastRenewal = DateTime.UtcNow;
+                    _lastRenewalLatency = _lastRenewal - requestStart;
                 }
                 else
                 {
                     LeaseId = await _lockBlob.AcquireLeaseAsync(_leaseTimeout, _instanceId);
+                    _lastRenewal = DateTime.UtcNow;
+                    _lastRenewalLatency = _lastRenewal - requestStart;
+
                     _traceWriter.Info($"Host lock lease acquired by instance ID '{_instanceId}'.");
 
                     // We've successfully acquired the lease, change the timer to use our renewal interval
                     SetTimerInterval(_renewalInterval);
                 }
             }
             catch (StorageException exc)
-            {
+            {                
                 if (exc.RequestInformation.HttpStatusCode == 409)
                 {
-                    ProcessLeaseError("Another host has an active lease.");
+                    // If we did not have the lease already, a 409 indicates that another host had it. This is 
+                    // normal and does not warrant any logging.
+
+                    if (HasLease)
+                    {
+                        // The lease was 'stolen'. Log details for debugging.
+                        string lastRenewalFormatted = _lastRenewal.ToString("yyyy-MM-ddTHH:mm:ss.FFFZ", CultureInfo.InvariantCulture);
```

Comment:
Curious - why would this info be useful to log? Not opposed to it if you think it would be useful.